QStreak/View Controllers/Record List/RecordListViewController.swift

```diff
@@ -23,6 +23,14 @@ class RecordListViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         viewModel.delegate = self
+
+        viewModel.recordsDidChange = { [weak self] result in
+            DispatchQueue.main.async {
+                self?.tableView.reloadData()
```

Comment:
Probably want to just reload only the rows that you've fetched instead of the entire table since this could potentially be a long list. Also, could take a look at prefetching. Good guide [here](https://www.raywenderlich.com/5786-uitableview-infinite-scrolling-tutorial).